src/buildstream/_yaml.pyx

```diff
@@ -389,6 +389,8 @@ yaml.RoundTripConstructor.add_constructor(u'tag:yaml.org,2002:timestamp',
 def prepare_roundtrip_yaml():
     yml = yaml.YAML()
     yml.preserve_quotes=True
+    # defaults to 80 if we don't set it
+    yml.width=9999
```

Comment:
Have we tried a negative value here ? It would seem a value of `-1` may be unlimited.

Otherwise `9999` seems like a perfectly sensible value.